shared/macros/10-bash.jinja

```diff
@@ -847,6 +847,45 @@ fi
 {{%- endmacro -%}}
 
 
+{{#
+    Enable pam_unix.so PAM module by using pam-auth-update.
+    This option is only recommended when pam-auth-update tool is available for the system.
+#}}
+{{%- macro bash_pam_unix_enable() -%}}
+conf_name=cac_unix
+if [ ! -f /usr/share/pam-configs/"$conf_name" ]; then
+    cat << EOF > /usr/share/pam-configs/"$conf_name"
+Name: Unix authentication
```

Comment:
Should we create a copy of `unix` (and add Conflicts and bump Priority) instead of hardcoding like this?